javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,90 @@
+
+let todoList=[]; // here is where the todo will be stored
+
+function getReply(command){
+    let name=""; 
+
+let split=command.split(" ");  // this line of code will create an array by dividing the command string into an array
+let firstWord=split[0]; // first word of the array
+let subject = split.slice(1).join(' '); // the rest of the array, disregarding the first
+let VerbToDo=split[1]; // second word of the array that will be displayed later
+
+switch(firstWord){  //switch case method
+    case "Hello":
+    name=subject;
+    return `Nice to meet you ${name}`;
+    break;
+    case "What":
+        if(subject === "is my name?"){
```

Comment:
switch case, and again 'if else' inside it. Please take a look at the approved homework of others.